Rules-page/rules.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://create-ui.netlify.app/css/components.css"
+    />
+    <link rel="stylesheet" href="/Rules-page/rules.css" />
+    <link rel="stylesheet" href="/variable.css" />
+    <title>Rules</title>
+  </head>
+  <body>
+    <div class="rules-container">
+      <div class="rules-text">
+        <h2 class="center-text mb-1">Rules</h2>
+        <p>This Quiz consists of 5 questions.</p>
+        <p class="my-half">Each right answer carries 2 points</p>
```

Comment:
I think rules can be made more viewable by providing then in ordered list. Other than that everything good

Reply:
ok